Makefile

```diff
@@ -232,3 +240,38 @@ $(SCALITY_EL7_REPODATA): $(SCALITY_EL7_RPMS) | $(PACKAGE_BUILD_CONTAINER)
 		--rm \
 		$(PACKAGE_BUILD_IMAGE) \
 		/entrypoint.sh buildrepo
+
+
+$(ISO_ROOT)/images/$(COREDNS_IMAGE_NAME)-$(COREDNS_IMAGE_VERSION).tar.gz: \
```

Comment:
Overall I do agree with this block, it's just that having "=" without space makes it less "Makefile stylish".
No big deal, just saying.
For reference, I guess you used this : https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html ?
Given the knowledge about Makefile overall, would be cool to include it in the commit message IMO.

Reply:
Bad bash habits...
It's fixed